src/github3/pulls.py

```diff
@@ -487,6 +488,8 @@ def merge(self, commit_message=None, sha=None, merge_method='merge'):
 
         :param str commit_message:
             (optional), message to be used for the merge commit
+        :param str commit_title:
+            (optional), message to be for merge commit title
```

Comment:
Nit: message to be **used** for **the** merge commit title

Reply:
Fixed.